public/wp-content/themes/glific/src/scss/style.scss

```diff
@@ -14,10 +14,6 @@ Text Domain: coloredcow
 
 @import "assets/bootstrap/bootstrap";
 
-body {
-    @extend .mt-26, .mt-xl-35;
```

Comment:
intentionally?

Reply:
what ?. I have define  styling in the dom